client/state/action-types.ts

```diff
@@ -625,9 +625,6 @@ export const PLUGIN_UPLOAD_COMPLETE = 'PLUGIN_UPLOAD_COMPLETE';
 export const PLUGIN_UPLOAD_ERROR = 'PLUGIN_UPLOAD_ERROR';
 export const PLUGIN_UPLOAD_PROGRESS = 'PLUGIN_UPLOAD_PROGRESS';
 export const PLUGINS_RECEIVE = 'PLUGINS_RECEIVE';
-export const PLUGINS_REQUEST = 'PLUGINS_REQUEST';
```

Comment:
Unrelated, but it feels off to maintain a `RECEIVE` action without a `REQUEST` one. This is a side effect of the fact that we use `react-query` for fetching, but not for state management.

Reply:
I didn't see a need to still keep the `PLUGINS_REQUEST` mostly because it wasn't used anymore. We still used the `PLUGINS_RECEIVE` one because the idea was to keep the

Reply:
I see 👍  Makes sense in the context of using `react-query` only to manage plugin requests as a first pass.